wordle-web/src/views/WordleView.vue

```diff
@@ -1,55 +1,54 @@
 <template>
-  <h1>Wordle Mind Bender</h1>
-
-  <GameBoard :game="game" @letterClick="addChar" />
-
-  <KeyBoard @letterClick="addChar" :guessedLetters="game.guessedLetters" />
-
-  <v-btn @click="checkGuess" @keyup.enter="checkGuess"> Check </v-btn>
-
-  <h2>{{ guess }}</h2>
-  <h3>{{ game.secretWord }}</h3>
+  <v-container>
+    <h1>Wordle Mind Bender</h1>
+
+    <GameBoard :game="game" @letterClick="addChar" />
+
+    <KeyBoard
+      :game="game"
+      :guessedLetters="game.guessedLetters"
+      @letter-click="addChar($event)"
+      @backspace="backspace"
+      @check-guess="checkGuess"
+    />
+    <AvailableWordsButton :wordle-game="game" :key="game.guesses.length" @guessChanged="setGuess" />
+    <h3>{{ game.secretWord }}</h3>
+  </v-container>
 </template>
 
 <script setup lang="ts">
 import { WordleGame } from '@/scripts/wordleGame'
-import { ref, reactive, onMounted, onUnmounted } from 'vue'
+import { ref, reactive } from 'vue'
 import GameBoard from '../components/GameBoard.vue'
 import KeyBoard from '../components/KeyBoard.vue'
-import type { Letter } from '@/scripts/letter'
+import { Letter } from '@/scripts/letter'
+import AvailableWordsButton from '@/components/AvailableWordsButton.vue'
 
 const guess = ref('')
 const game = reactive(new WordleGame())
 console.log(game.secretWord)
 
-onMounted(() => {
-  window.addEventListener('keyup', keyPress)
-})
-onUnmounted(() => {
-  window.removeEventListener('keyup', keyPress)
-})
-
 function checkGuess() {
```

Comment:
I think you should check for length of guess, and probably do some regex operations to filter non-letter chars here.